libnmstate/nm/ipv6.py

```diff
@@ -47,17 +47,30 @@ def get_info(active_connection):
     return info
 
 
+def is_link_local(address):
+    return iplib.is_subnet_of(address, iplib.IPV6_LINK_LOCAL_NETWORK)
+
+
 def create_setting(config):
     setting_ip = nmclient.NM.SettingIP6Config.new()
     if config and config.get('enabled') and config.get('address'):
         setting_ip.props.method = (
             nmclient.NM.SETTING_IP6_CONFIG_METHOD_MANUAL)
+        has_static_addr = False
         for address in config['address']:
+            if is_link_local("%s/%s" % (address['ip'],
```

Comment:
Not sure, but shouldn't we allow link-local addresses as long as they are not auto-generated?